src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/src/ansys/solutions/{{cookiecutter.__solution_name_slug}}/ui/views/design_table_view.py

```diff
@@ -146,3 +149,20 @@ def update_view(n_intervals, pathname):
             raise PreventUpdate
     else:
         raise PreventUpdate
+
+
+@callback(
+    Output("dummy-table-output", 'children'),
```

Comment:
I see in app.py we are using the NoOutputTransform from dash extensions enrich, which should mean you don't need the hidden div added on line 59 and this dummy Output element.

Reply:
Will update the callback after testing.

Reply:
[91df52c4259acebdbb588f48f2e106c45bc8ccb8](https://github.com/ansys/ansys-templates/pull/414/commits/91df52c4259acebdbb588f48f2e106c45bc8ccb8)  Updated in this commit